xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.h

```diff
@@ -222,6 +222,7 @@ class CRenderManager
 
   struct SPresent
   {
+    SPresent();
```

Comment:
Instead of using a constructor you can init the members by assignemnt directly below.